### PR TITLE
FEATURE - About page

### PR DESCRIPTION
**Pull request by manuelosorio**
[FEATURE - /about page](https://github.com/TampaDevs/tampadevs/issues/9) 
**About Page:**
-   The Story of Tampa Devs
-   List of Event Organizers
-   Link to Tampa Devs Sponsors
-   Add a Gallery of Past Events
-   Update the people.json data file
-   Add a JSON file for past events.

**Util:**
- Create SCSS mixin to use with media queries for variable margins 
- Create SCSS mixin for custom Media queries widths

### CODE REVIEW

File: src/_includes/layouts/about.njk [hunk 1]
```diff
@@ -1,39 +1,231 @@
...
+      </li>
+      <li>
+        <a href="https://www.tampadevs.com/blog">
+          BLOG
+        </a>
+      </li>
+    </ul>
+  </div>
+  <br/>
+  <div class="td-footer__social">
```
Comment (vincentntang) [inline src/_includes/layouts/about.njk]: could you move the footer code in `base.njk`  into `_includes/blocks/footer.njk`?

and then insert that same block of code into `about.njk` and `base.njk`

as of right now the footer is different on the `/about` page compared to rest of site

**Comment by vincentntang** [comment]
could you move the footer code in `base.njk`  into `_includes/blocks/footer.njk`?

and then insert that same block of code into `about.njk` and `base.njk`

as of right now the footer is different on the `/about` page compared to rest of site

**Comment by vincentntang** [comment]
image

^could you use this logo for brooksource?

also delete ashley furniture and exzeo from the list of sponsors

**Comment by manuelosorio** [comment]
Made requested changes and ready for another code review. @vincentntang

File: src/_assets/scss/utility/_mixin-sizing.scss [hunk 2]
```diff
@@ -22,51 +22,56 @@
...
-  }
-  .py--#{$i} {
-    padding-top: $i * $spacer !important;
-    padding-bottom: $i * $spacer !important;
-  }
-}
+
+
+
+@mixin testMargin {
```
Comment (vincentntang) [inline src/_assets/scss/utility/_mixin-sizing.scss:28]: could you re-enable these mixins? they aren't test margins, there actually being used in the card component on the homepage for meetup/slack

File: src/_includes/layouts/base.njk [hunk 3]
```diff
@@ -57,161 +57,11 @@
...
-    </p>
-    <div class="td-nav footer-source">
-      <a href="https://www.github.com/tampadevs/tampadevs" target="_blank" rel="noopener">View Source</a>
-    </div>
+    {% include "blocks/newsletter.njk" %}
+    {% include "blocks/footer.njk" %}
+    <script src="/_assets/js/script.js?v={{ data.buildRef }}"></script>
   </body>
-</html></footer><script src="/_assets/js/script.js?v={{ data.buildRef }}"></script><script>
-AOS.init();</script></body></html>
```
Comment (vincentntang) [inline src/_includes/layouts/base.njk]: this probably needs to be added back in, its for the animation on the card component for the "Join meetup" and "Join slack" cards further down the page

**Comment by vincentntang** [comment]
aos animation needs to be added back in thats currently in production on https://tampadevs.com. 

Should look like this:


Animate on scroll docs found here:
http://michalsnik.github.io/aos/

**Pull request closed without merging**